backend/src/main/kotlin/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -35,4 +44,33 @@ class CardMutationService {
         }
         return true
     }
+
+    @GraphQLDescription("Activate a digital EAK")
+    fun activateCard(
+        project: String,
+        card: CardGenerationModel,
```

Comment:
I think we should not reuse the CardGenerationModel here.
Instead, we should ask only for the things we need: cardInfoHashBase64 and activationSecretBase64.